### PR TITLE
[IMP] activities: updating for 17.0

### PR DESCRIPTION
**Pull request by larm-odoo**
Current doc has old pictures of previous versions of Odoo. Updating images and any directions that have changed.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @Felicious - this is ready for a peer review. Thanks!

File: content/applications/essentials/activities.rst
Comment (Felicious) [inline content/applications/essentials/activities.rst:281]:
```suggestion
  - :guilabel:`Call` or :guilabel:`Meeting`: if selected, users have the option to open their
```

File: content/applications/essentials/activities.rst
Comment (Felicious) [inline content/applications/essentials/activities.rst:288]:
```suggestion
     Available activity types vary based on the installed applications in the database.
```

File: content/applications/essentials/activities.rst
Comment (Felicious) [inline content/applications/essentials/activities.rst:296]: This line renders kinda funky :upside_down_face: 
image

How about trying to put a blank line between this line and 293 right above?

```suggestion

- :guilabel:`Default User`: select a user from the drop-down menu to automatically assign this
   activity to the selected user when this activity type is scheduled. If this field is left blank,
   the activity is assigned to the user who creates the activity.
```

File: content/applications/essentials/activities.rst
Comment (Felicious) [inline content/applications/essentials/activities.rst:321]: This is a very helpful field! I think it's a little hard to conceptualize generally, and I suggest keeping the first sentence:
```suggestion
- :guilabel:`Schedule`: configure when the next activity is suggested or triggered.
```
Then, put the rest of the paragraph you wrote for this field in an admonition block :smile:, something like:

.. example::
   To schedule a follow-up call three days after the first call, do this...

**Review by Felicious — APPROVED** [review]
Hello @larm-odoo ! I wanted to say great job on this super detailed documentation about scheduling activities! 🌟

I had a few rewording suggestions to help shorten some sentences a bit. Feel free to pick and choose what works best for you! Keep up the awesome work! 💪

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:262]: I htink I will leave it because the section is so short,  s it doesn't feel like too much 'extra' ifo (and it applies to all the forms, so I thik it's OK to keep it in here).

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:321]: I prefer to keep the generic altogether so the steps aren't sepaarated. What I did was space itout, so visually it wasn't a huge chunk of text, so I added a few paragraph returns (each step has it's own paragraph now).

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:296]: I saw that in my make-fast but couldn't figure out the issue. A extra space didn't help either.  Turns out, I added too much space at the end of the line (early line break) and THAT'S why it did that! Who knew!?

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:58]: The only reason I don't want to do that, is because it will only look like a clock if nothing is scheduled. So if a user goes in and see's a different icon, i want them to be sure of what ti is. That's why I link the 'activity icon' phrase every time to the top portion that lists how they can appear. So while I understad your logic here, I have to politely disagree. I am thinking baout users reading this, interacting in a database that has things scheduled, and don't want them to think they can't schedule osmething because the icon is wrong.

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:168]: I moved the first sentence to the end, and rephrased it a bit, and added more spaces as well.

**Comment by larm-odoo** [comment]
Hey @ksc-odoo - this is ready for a final doc review!

**Comment by larm-odoo** [comment]
Just wanted to say thank you for the review, @Felicious! I resolved all the comments, and did most of the changes. In some places where I did not, or did a combo of your edits plus mine, I explained why in a comment.

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:197]:
```suggestion
If any activities are scheduled, the number of activities appear in a red bubble on the
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:209]:
```suggestion
   These requests appear in two separate lists in the all activities drop-down menu: one labeled `Time
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:221]:
```suggestion
:guilabel:`Request a Document` appears. Click :guilabel:`Request a Document`, and a
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:239]:
```suggestion
When all the fields are completed, click :guilabel:`Request` to send the document request.
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:251]:
```suggestion
:menuselection:`Settings app --> Discuss section --> Activities setting --> Activity Types`.
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:267]:
```suggestion
activity type form loads.
```
technically, this isn't what the page is titled on the UI...so a GUI is not applicable in this case, in my opinion

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:269]:
```suggestion
Make any desired changes to the activity type form. The form automatically saves, but it
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:270]:
```suggestion
can be saved manually at any time by clicking the :guilabel:`Save manually` option, represented by a
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:276]:
```suggestion
To create a new :ref:`activity type <activities/types>`, click :guilabel:`New` from the :guilabel:`Activity Types` page, and a blank
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:277]:
```suggestion
activity type form loads.
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:313]:
```suggestion
- :guilabel:`Keep Done`: tick this checkbox to keep activities that have been marked as `Done` visible in the
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:323]:
```suggestion
- :guilabel:`Chaining Type`: select either :guilabel:`Suggest Next Activity` or :guilabel:`Trigger
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:329]:
```suggestion
  the activity to recommend or schedule as a follow-up task to the activity type.
```

File: content/applications/essentials/activities.rst
Comment (ksc-odoo) [inline content/applications/essentials/activities.rst:338]:
```suggestion
  Lastly, using the drop-down menu, select whether the activity is scheduled or triggered either
```

**Review by ksc-odoo — APPROVED** [review]
hey @larm-odoo -- just finished my Final Review. Great work! Just have a handful of comments that require your attention. But, once you implement the necessary adjustments, you can feel free to tag this for Tech Review. Thanks! 👍

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:267]: I was torn, because the name doesn't appear when doing this specific flow, BUT the form is identical/is the same as the "acivity type" form- which does say those words when mking an activity in another way. Since it was the same fom, I used the GUI-BUT I completely understand your logic. I admit, i do not understand why they didn't have the title for the form appear here when it is clearly the same form.

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:174]: It originally was but was recommended to move it down =D

Comment (larm-odoo) [reply content/applications/essentials/activities.rst:174]: I made this an important admonition block, so this way it's highlighted, and it's at the end. The other peer review just suggested to keep all the instructions togther and not break it up with the note- which sounded good to me.

**Comment by larm-odoo** [comment]
Hey @samueljlieber - this is ready for a tech review whenever you have a moment. Thanks!

File: content/applications/essentials/activities.rst
Comment (samueljlieber) [inline content/applications/essentials/activities.rst:327]: Suggestion to explain that selecting an option in this field, affects the display of other fields
```suggestion
- :guilabel:`Chaining Type`: select either :guilabel:`Suggest Next Activity` or  :guilabel:`Trigger
  Next Activity` from the drop-down menu. Depending on the selected option, either the
  :guilabel:`Suggest` or :guilabel:`Trigger` field is displayed.
```

File: content/applications/essentials/activities.rst
Comment (samueljlieber) [inline content/applications/essentials/activities.rst:329]: Suggestion to place this in a note and avoid "This"
```suggestion
  .. note::
     The :guilabel:`Chaining Type` field does **not** appear if :guilabel:`Upload Document` is
     selected for the :guilabel:`Action`.
```

File: content/applications/essentials/activities.rst
Comment (samueljlieber) [inline content/applications/essentials/activities.rst:349]: Update these paths to be relative please
```suggestion
   - :doc:`../productivity/discuss`
   - :doc:`../productivity/discuss/team_communication`
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @larm-odoo, nice job on updating the content on this activities essentials doc! After reading through for my technical review I have a handful of suggestions. Please see below and let me know your thoughts!

Also, I think the commit message could be more descriptive of what this PR does, since images are not the only updated content. Something like:
```
[IMP] activities: updating for 17.0
```
Please tag me for another quick look once these have been addressed thank you!

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:23]: I agree- I went with 'importnat' because since this icon could look one of 7 ways, I wanted it clear that it's a changing icon. But I agree with you about an important block being jarring at the begining of a doc =)

I just reformatted what I have since it explained what was happening, and I thought it makes sense.

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:225]: You are so right!

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:160]: It seees odd to have 2 images together, one after the other, so I removed the one that calls out the "Planned activities' and "today", figuring that at least is intuitive, so it's not super necessary to include.

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:59]: I disagree only because this is the essentials doc, and the icons are not always called out/are not always intuitive. I know you cna read them when you mouse over it, but I know I wouldn't pick out this icon on my own. I do like keeping this in, since I think it is of value.

**Comment by larm-odoo** [comment]
Thanks @samueljlieber! I originally made the PR for ONLY the images but then determined the info was wrong and needed to be changed, so I coudln't leave it at just the images =D. I made all changes requested- ready for another look.

File: content/applications/essentials/activities.rst
Comment (samueljlieber) [inline content/applications/essentials/activities.rst:7]: Suggestion to give this custom anchor a more descriptive name:
```suggestion
.. _activities/icon-display:
```
Or remove it since it is not used in this doc (however it is helpful to leave it for future references 🙂 )

File: content/applications/essentials/activities.rst
Comment (samueljlieber) [inline content/applications/essentials/activities.rst:170]: Indent text in important admonition one more space please :)
```suggestion
     The :guilabel:`Open Calendar` button **only** appears if the :guilabel:`Activity Type` is set
     to either :guilabel:`Call` or :guilabel:`Meeting`.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo, thank you for implementing most of my suggestions! I appreciate you considering them, and I think this essentials doc is looking pretty great! I have just a few more corrections that I caught on my second pass, please take care of these before merging. Approving now and delegating to you :)

The following images are unused and should be removed:
```
activities/new-activity-type.png
activities/schedule-activity-view.png
activities/schedule-activity-view-menu.png
activities/schedule-recommended-activity.png
activities/triggered-activities.png
```
..
@robodoo delegate=larm-odoo

File: content/applications/essentials/activities.rst
Comment (larm-odoo) [reply content/applications/essentials/activities.rst:7]: I was just checking that- i don't use this since I don't reference this anymore, so it's getting the ax!

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
